packages/bot-web-ui/src/components/dashboard/quick-strategy/quick-strategy-components/container.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import { TQuickStrategyProps, TSymbolItem } from '../quick-strategy.types';
+import { QStrategyForm, MarketOption, TradeTypeOption } from '.';
+
+const QStrategyContainer = (props: TQuickStrategyProps) => {
+    const {
+        symbol_dropdown,
+        trade_type_dropdown,
+        active_index,
+        description,
+        createStrategy,
+        duration_unit_dropdown,
+        types_strategies_dropdown,
+        getSizeDesc,
+        initial_values,
+        is_onscreen_keyboard_active,
+        is_mobile,
+        is_stop_button_visible,
+        onChangeDropdownItem,
+        onChangeInputValue,
+        onHideDropdownList,
+        onScrollStopDropdownList,
+        selected_symbol,
+        selected_trade_type,
+        setCurrentFocus,
+        selected_duration_unit,
+        selected_type_strategy,
+        setActiveTab,
+    } = props;
+
+    const symbol_dropdown_options = React.useMemo(
+        () =>
+            symbol_dropdown
+                .map((symbol: TSymbolItem, idx) => ({
+                    component: <MarketOption key={idx} symbol={symbol} />,
+                    ...symbol,
+                }))
+                .filter(option => option.group !== 'Cryptocurrencies'), // Until Crypto enabled for Dbot
+        [symbol_dropdown]
+    );
+
+    const trade_type_dropdown_options = React.useMemo(
```

Comment:
since the props in these drop downs are changing frequently, is it a good idea to use react.memo? just wanna make sure

Reply:
Duplicate previous comment